app/views/articles/search/_meta.html.erb

```diff
@@ -8,7 +8,7 @@
 <meta property="og:title" content="The DEV Community" />
 <meta property="og:image" content="<%= SiteConfig.main_social_image %>">
 <meta property="og:description" content="<%= SiteConfig.community_description %>">
-<meta property="og:site_name" content="The DEV Community" />
+<meta property="og:site_name" content="<%= ApplicationConfig["COMMUNITY_NAME"] %> Community" />
```

Comment:
To save typing (well, you've already typed, but still...): what if you define a `community_name` method in `ApplicationHelper` and return the whole blob `ApplicationConfig["COMMUNITY_NAME"]` ?

Reply:
I like it!

Reply:
Turns out we already have a community_qualified_name helper :) Replaced it with that :)